R/queue.R

```diff
@@ -75,24 +75,30 @@ Queue <- R6::R6Class("Queue", # nolint
     #' @description
     #' Gets status of packet run
     #'
-    #' @param task_id Id of redis queue job to get status of.
+    #' @param task_ids Task ids  to get status of.
+    #' @param include_logs Whether to include logs in response or not
     #' @return status of redis queue job
-    get_status = function(task_id) {
-      if (!rrq::rrq_task_exists(task_id, controller = self$controller)) {
-        porcelain::porcelain_stop("Job ID does not exist")
+    get_status = function(task_ids, include_logs = TRUE) {
+      valid_task_ids <- task_ids[rrq::rrq_task_exists(task_ids, controller = self$controller)]
+      invalid_task_ids <- setdiff(task_ids, valid_task_ids)
+      if (length(invalid_task_ids) > 0) {
+        cli::cli_warn(paste("Job ids", paste(invalid_task_ids, collapse = ", "), "do not exist in the queue"))
```

Comment:
```suggestion
        porcelain::porcelain_stop(paste("Job ids:", paste(invalid_task_ids, collapse = ", "), "do not exist in the queue"))
```

Reply:
apologies this mightve been my fault with the suggestion i gave cos i accidentally used cli::cli_warn, still gotta use porcelain stop if you want to actually send back the error

Reply:
done